Formula/global.rb

```diff
@@ -4,6 +4,7 @@ class Global < Formula
   url "https://ftpmirror.gnu.org/global/global-6.5.6.tar.gz"
   mirror "https://ftp.gnu.org/gnu/global/global-6.5.6.tar.gz"
   sha256 "122f9afa69a8daa0f64c12db7f02981fe573f51a163fa3829ed4f832cd281505"
+  revision 1
```

Comment:
I wasn't sure whether exposing a new option warranted/required adding a `revision`. I'm starting to suspect not, but I'll leave this for now to get some outside feedback.